### DISCUSSION TITLE
function call parameters内部的参数类型必须是字符串么

### DISCUSSION THREAD

**Discussion by wanghq09** [zai-org/ChatGLM3#729] [Q&A]
参数parameters本身是否可以为list，parameters内部的参数类型是否可以是str之外的其他类型，这里没有看到类型的定义位置
另外，看注释说返回值可以是各种类型，是这样么

**Comment by si1mon2** [comment]
可以是list，也可以是int或者float，把Annotated里面的str改成对应的类型就可以。返回值也可以是list，int之类的值。
